epoch.go

```diff
@@ -980,7 +1163,7 @@ func (e *Epoch) startRound() error {
 	return e.handleBlockMessage(msgsForRound.proposal, leaderForCurrentRound)
 }
 
-func (e *Epoch) doProposed(block Block, voteFromLeader Vote) error {
+func (e *Epoch) doProposed(block Block, voteFromLeader Vote, _ NodeID) error {
```

Comment:
Why is this argument added if it is never used?

Reply:
Removed it.